examples/Demo/FluentUI.Demo.Client/Documentation/Components/Image/Examples/FluentImageCustomStyling.razor.css

```diff
@@ -0,0 +1,4 @@
+﻿::deep .fluent-image-item{
```

Comment:
Can you use a style like this to make a border size more visible? `border: 3px dashed dodgerblue;`